torch/quantization/quantize_fx.py

```diff
@@ -0,0 +1,149 @@
+from .fx import Fuser  # noqa: F401
+from .fx import Quantizer  # noqa: F401
+
+def fuse_fx(graph_module, inplace=False):
+    fuser = Fuser()
+    return fuser.fuse(graph_module, inplace)
+
+def _prepare_fx(graph_module, qconfig_dict, inplace, is_dynamic):
+    quantizer = Quantizer()
```

Comment:
do we need a stateful object for this?  Can it be stateless?

Reply:
There are some shared functions, I think it can be further refactored as functions, but this can be done in future prs